connector_oxigesti/models/product_pricelist_item/binding.py

```diff
@@ -7,6 +7,24 @@
 from odoo.exceptions import ValidationError
 
 
+class Pricelist(models.Model):
+    _inherit = "product.pricelist"
+
+    # TODO: Review use deleter to mark items as deprecated
+    def unlink(self):
+        for rec in self:
+            if rec.item_ids.oxigesti_bind_ids:
+                raise ValidationError(
+                    _(
+                        "You can't delete a pricelist that has been exported "
+                        "the product prices by customer to Oxigesti. If you "
+                        "want to delete it, you must first delete the items "
+                        "of the pricelist."
```

Comment:
Add to the message what pricelist is `rec.name`, it will be useful when the user deletes the pricelists massively from tree view